demo.md

```diff
@@ -62,24 +63,24 @@ The following requirements should be met:
 * The code should be production grade.
 * It should compile and run.
 
-### 2. Already written project
+### 2. Previously written project
 
-We would be happy if you would submit a project you already have (for instance, a demo project for another company). Still, the project **must** obey the following requirements:
+We would be happy if you would submit a project you already have (for instance, a demo project for another company). Still, the project **must**:
 
 * Use Swift 3.0 or above.
-* It should have at least two distinct network calls.
-* It should parse the network response and present the information to the user.
-* It should have some sort of persistence mechanism.
-* It should compile and run.
-* It should have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
+* Have two distinct network calls.
+* Parse the network response and present the information to the user.
+* Have some sort of persistence mechanism.
+* Compile and run.
+* Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
 
-If you have a project with these requirements, then perfect! Please ensure that you also have a description of what the project does, in order to give us some context.
+If you have a project with these requirements, then perfect! Please ensure that you also have a description of what the project does in order to give us some context.
```

Comment:
I disagree with removing that comma. "in order to give us some context" is a secondary sentence that is not directly tied to the preceding sentence and, therefore, should be preceded by a comma.